### PR TITLE
[chart, table] marked these packages as deprecated

### PR DESCRIPTION
**Pull request by freeyoungstrong**
## Description

Marked packages `@semcore/chart` and `@semcore/table` as deprecated and added recommendations to use `@semcore/d3-chart` and `@semcore/data-table` instead.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: semcore/input-tags/src/InputTags.tsx [hunk 1]
```diff
@@ -17,8 +17,9 @@ export interface IInputTagsProps extends Omit<IInputProps, 'size'>, IScrollAreaP
    * @default m
    */
   size?: InputTagsSize;
-  /** @deprecated use `onAppend` instead */
-  /** Event is called when tag needs to be added */
+  /** Event is called when tag needs to be added
+   * @deprecated use `onAppend` instead
```
Comment (phytonmk) [inline semcore/input-tags/src/InputTags.tsx:21]: Why this file was affected?

**Review by phytonmk — CHANGES_REQUESTED** [review]
**I was waiting for that almost for twho years!**

But, two things:
1. `input-tags` was affected but should be not
2. Changelogs updates are also needed

File: semcore/input-tags/src/InputTags.tsx (hunk 1 above)
Comment (freeyoungstrong) [reply semcore/input-tags/src/InputTags.tsx:21]: I just walked near this file and noticed that the description of the prop was written incorrectly I mean users don't see that it was deprecated and need to use another function
Screenshot 2022-09-15 at 18 30 04

Comment (freeyoungstrong) [reply semcore/input-tags/src/InputTags.tsx:21]: I just fixed description

Comment (freeyoungstrong) [reply semcore/input-tags/src/InputTags.tsx:21]: Screenshot 2022-09-15 at 18 31 28

Comment (freeyoungstrong) [reply semcore/input-tags/src/InputTags.tsx:21]: but yes I understand thta in this case commit message and these changes are not the same. anyway I can return everything as it was :)

**Pull request merged**